TP/css/style.css

```diff
@@ -92,10 +92,10 @@ article{
 aside{
     background-color: #359C95;
     border-left: 1px solid rgb(51,51,51);
-    position: absolute;
     padding-top: 20px;
     right: 0px;
     width: 25%;
+    float: right;
```

Comment:
Lo flote a la derecha porque si dejas la ``position: absolute`` el footer the queda arriba por encima del aside.